docs/build.yml

```diff
@@ -12,3 +12,5 @@ dependencies:
   - docutils
   - mu-notedown
   - flake8
+  - torch
+  - neotime
```

Comment:
can you explain why neotime is necessary as a dependency? This package appears to have 2 stars on github total.

Reply:
neotime seems just necessary for the Date preprocessing stuff, so let's remove it.